OPAL/si/src/main/scala/org/opalj/fpcf/scheduling/MaximumPhaseScheduling.scala

```diff
@@ -98,7 +97,39 @@ abstract class MultiplePhaseScheduling extends SchedulingStrategy {
             computePhase(ps, phaseAnalyses, remainingAnalyses)
         }
 
-        schedule
+        if(config.getBoolean(DisableCleanup)) calculateDeletions(schedule, ps, config) else schedule
+    }
+
+    private def calculateDeletions[A](schedule: List[PhaseConfiguration[A]], ps: PropertyStore, config: Config): List[PhaseConfiguration[A]] = {
+        def getSetOrNilForProperty(propertyKey: String) = {
+            if (!config.hasPath(propertyKey)) Nil
+            else {
+                val rawValue = config.getString(propertyKey).trim
```

Comment:
Why all the `trim`s? If the format of the config is properly speficied, there should be nothing to trim.